app/components/trainees/assessment_details/view.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= render_component SummaryCard::View.new(
+  title: "Assessment details",
```

Comment:
Should this be 'Outcome details'? General question about which prototype we should be working from!

Reply:
probably the "real" prototype :) https://github.com/DFE-Digital/register-trainee-teachers/pull/235/commits/9672f12c607a2789fff0ea03eabc16d87a320b7d